tests/conftest.py

```diff
@@ -229,7 +229,6 @@ def event_loop(request):
     asyncio_logger = logging.getLogger("asyncio")
     asyncio_logger.setLevel("WARNING")
     asyncio_logger.addHandler(logging.StreamHandler())
-    loop.set_debug(True)
```

Comment:
It could be nice to have an option that turns this on/off easily if we need it, but I don't think that should block this PR.

Reply:
good call, updated to be a unit testing setting